### PR TITLE
add custom user agent, fixes #112

### LINKED ISSUE #112: Set user agent for bgp.tools table.txt pull
Hi there!

bgp.tools has made the table.txt/jsonl endpoint official now. But with a requirement for the user agent to not be a default one. Right now irrexplorer uses `Python/3.9 aiohttp/3.7.4`. Could this be changed to something like "NLNOG - irrexplorer" or something similarly descriptive? 

I've banned default library user agents right now, but special caused the one above that irrexplorer is using until you can hopefully set a custom one!

Thanks again for this great tool

### PR DESCRIPTION
**Pull request by teunvink**
Add a custom 'IRR explorer' user agent to all HTTP requests done by the back end, as requested in #112.

### CODE REVIEW

File: irrexplorer/backends/common.py [hunk 1]
```diff
@@ -48,7 +48,7 @@ async def retrieve_url_text(url: str):
     Retrieve `url` and return the text in the response.
     Raises ImporterException if the response status is not 200.
     """
-    async with aiohttp.ClientSession() as session:
+    async with aiohttp.ClientSession(headers={"User-Agent": "IRR explorer"}) as session:
```
Comment (job) [inline irrexplorer/backends/common.py:51]: IRRExplorer? :)

Comment (teunvink) [reply irrexplorer/backends/common.py:51]: I actually took the name from the webpage, but I'm more than fine with that as well.

Comment (teunvink) [reply irrexplorer/backends/common.py]: Changed it to IRRExplorer now.

**Comment by teunvink** [comment]
You should now see a `IRRExplorer` user agent string. Fixed both in code and on https://irrexplorer.nlnog.net.

**Pull request merged**